build.sbt

```diff
@@ -263,6 +264,7 @@ lazy val catsInterop = project
       else Seq(compilerPlugin(("org.typelevel" %% "kind-projector" % "0.13.2").cross(CrossVersion.full)))
     } ++ Seq(
       "org.typelevel" %% "cats-effect"      % catsEffect3Version,
+      "co.fs2"        %% "fs2-core"         % fs2Version,
```

Comment:
This dependency is made `Optional` in the `cats-interop` module. Should we make it optional here too?

Reply:
Yeha we could do that.

Reply:
To be honest I think we're okay with making this a required dependency. Http4s depends on fs2 as well, and the likelihood of a cats-effect app not having dependency on fs2 is very very low